apollo-gradle-plugin/testProjects/mismatchedVersions/settings.gradle.kts

```diff
@@ -1,2 +1,10 @@
 include(":module")
-rootProject.name = "mismatchedVersions"
\ No newline at end of file
+rootProject.name = "mismatchedVersions"
+
+dependencyResolutionManagement {
+  versionCatalogs {
+    create("libs") {
+      from(files("../../../gradle/libs.versions.toml"))
+    }
+  }
+}
```

Comment:
A bit unfortunate that we have to copy paste this block all over the place now but there's no silver bullet 🤷